x-pack/plugins/lists/public/exceptions/validate.ts

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
NOTE: When I tried to import these functions that we are already importing from SIEM, it would not build. Is there a way around this? Can we move these functions to the common folder in SIEM?

Reply:
Yes, it is coming in this PR right now:
https://github.com/elastic/kibana/pull/69296

Reply:
Thank you!